internal/config/config.go

```diff
@@ -166,6 +167,42 @@ func registerFlags() {
 	fs.StringArray(
 		PrometheusTargetsKey, []string{}, "The target URI(s) of Prometheus endpoint(s) for metrics collection.",
 	)
+	fs.String(
+		CommandServerHostKey, "", "The target hostanme of command server endpoint for command and control.",
+	)
+	fs.Int32(
+		CommandServerPortKey, 0, "The target port of command server endpoint for command and control.",
+	)
+	fs.String(
+		CommandServerTypeKey,
+		"grpc",
+		"The target protocol (grpc or http) command server endpoint for command and control.",
```

Comment:
Customer-facing string, so nitpicking here.
```suggestion
		"The target protocol (gRC or HTTP) command server endpoint for command and control.",
```